pushover/README.md

```diff
@@ -0,0 +1,85 @@
+# Pushover  
+
+Pushover is one out of many push services, which is compatible with Android, IOS and Windows Clients. This plugin make it possible to push messages to your clients from smarthomeNG.
+  
+## Requirements
+  
+### Python libraries  
+* requests - [install instructions](http://docs.python-requests.org/en/latest/user/install/#install "http://docs.python-requests.org/en/latest/user/install/#install")
```

Comment:
Do we need to mention the requirements here, when we have them listed in the requirements file?